starter-code/index.js

```diff
@@ -1,17 +1,36 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length++;
+    this.items.sort((a, b) => (a - b));
+  }
 
-  get(pos) {}
+  get(pos) {
+    throw new Error("OutOfBounds");
+  }
 
-  max() {}
+  max() {
+    throw new Error("EmptySortedList"); 
+  }
 
-  min() {}
+  min() {
+    return this.items.sort((a, b) => (a - b))[0]; 
```

Comment:
here you should have used the Math.min
```js
return Math.min(...this.items)
```

It still needs to check for the bad cases, though